Libraries/LibELF/ELFLoader.h

```diff
@@ -17,6 +17,8 @@ class ELFLoader {
     ~ELFLoader();
 
     bool load();
+    Optional<String> interpreter();
```

Comment:
This should be const since it does not mutate the internal state of the `ELFLoader`.

Reply:
True. I ended up finding the interpreter a different way, so this is OBE.